spec/controllers/people_controller_spec.rb

```diff
@@ -455,7 +455,7 @@
       end
 
       it "posts include reshares" do
-        reshare = @user.post(:reshare, :public => true, :root_guid => FactoryGirl.create(:status_message, :public => true).guid, :to => alice.aspect_ids)
+        reshare = @user.post(:reshare, :public => true, :root_guid => FactoryBot.create(:status_message, :public => true).guid, :to => alice.aspect_ids)
```

Comment:
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
- Layout/LineLength: Line is too long. [152/120]